samples/drivers/led/led_strip/README.rst

```diff
@@ -68,7 +68,7 @@ modified by changing the :kconfig:option:`CONFIG_SAMPLE_LED_UPDATE_DELAY`.
 Then build and flash the application:
 
 .. zephyr-app-commands::
-   :zephyr-app: samples/drivers/led_strip
+   :zephyr-app: samples/drivers/led/led_strip
```

Comment:
Don't you think we should put `led_strip` at the same level than `led` ?
```
samples/drivers/led
samples/drivers/led_strip
```
After all `led_strip` is a distinct subsystem, with its own API.  Also, this would mirror the `drivers` tree:
```
drivers/led
drivers/led_strip
```

Reply:
Maybe, yes. My reasoning was based on how things are presented in: https://docs.zephyrproject.org/latest/hardware/peripherals/led.html , i.e. everything under LED.

Reply:
Makes sense.

Reply:
I am happy to do a follow-up PR but hopefully this is good enough for a first pass at cleaning things up. 
I just rebased to address a conflict in `redirects.py`

Reply:
Would vote for led_strip being at the same level as led not inside of it, they are different drivers